starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,53 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
+
+ Chronometer.prototype.startClick = function () {
+  var that = this;
+  this.intervalId = setInterval(function(){
+     that.currentTime += 1;
+     that.setTime();
+  },1000);
+ };
+
+Chronometer.prototype.setMinutes = function () {
+  this.currentTimeMin = this.currentTime / 60;
+  return Math.floor(this.currentTimeMin);
```

Comment:
Mejor cread las propiedades arriba y luego las llamáis en los métodos si tenéis que cambiarlas. En este caso, crear this.currentTImeMin junto con las otras 2 propiedades que tiene Chronometer.